project/plugins.sbt

```diff
@@ -21,3 +21,5 @@ addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.2")
 
 addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-stamp" % "6.1.0")
```

Comment:
We're using `gitStampSettings` in `build.sbt` with upgrades we need to specify this manually to have access to those settings. Not sure that we need it